tests/integration-tests/conftest.py

```diff
@@ -189,7 +190,7 @@ def _cluster_factory(cluster_config):
         cluster = Cluster(
             name=request.config.getoption("cluster")
             if request.config.getoption("cluster")
-            else "integ-tests-" + random_alphanumeric(),
+            else "integ-tests-" + random_alphanumeric() + request.config.getoption("stackname_suffix"),
```

Comment:
minor: I think we should add a `-` here before concatenating the suffix

Reply:
I think the suffix should contain the - 
Let's discuss it